main/src/ca/uwaterloo/flix/language/phase/typer/PrimitiveEffects.Classes.json

```diff
@@ -3,50 +3,11 @@
     "java.lang.Boolean": "",
     "java.lang.Byte": "",
     "java.lang.Character": "",
-    "java.lang.Character$Subset": "",
```

Comment:
Why this change?

Reply:
So all the remaining annotations are required for std lib compilation?

Reply:
> So all the remaining annotations are required for std lib compilation?

Yes